packages/cli/src/activate.ts

```diff
@@ -18,6 +18,11 @@ import CLIUIInstance from "./userInteraction";
 
 export default async function activate(rootPath?: string): Promise<Result<FxCore, FxError>> {
   if (rootPath) {
+    AzureAccountManager.setRootPath(rootPath);
+    const subscriptionInfo = await AzureAccountManager.readSubscription();
+    if (subscriptionInfo) {
+      await AzureAccountManager.setSubscription(subscriptionInfo.subscriptionId);
+    }
     const subscription = await getSubscriptionIdFromEnvFile(rootPath);
```

Comment:
Is this line still needed?

Reply:
Have removed old read sub logic, ```getSubscriptionIdFromEnvFile``` only read subscriptionId, ```getSubscriptionInfoFromEnv``` read subscription and tenant information.